src/Codeception/Module/Symfony.php

```diff
@@ -22,6 +22,10 @@
  *
  * * Maintainer: **raistlin**
  * * Stability: **stable**
+ * 
+ * ## Compatibility
+ *
+ * This module is compatible with all other Codeception modules, except WebDriver and the modules for other PHP frameworks.
```

Comment:
1. OK, so how many parts do exist? I see `services` here http://codeception.com/docs/modules/Symfony#Parts and you're telling me `orm`. Anything else?

1. And is it true for all parts that their purpose is compatibility with WebDriver and PhpBrowser? Or do some of them have a different use case?

1. And can you load more than one part? If yes, how's the syntax?